apps/api/src/services/library-traffic.ts

```diff
@@ -0,0 +1,27 @@
+import type { libraryTrafficQuerySchema } from "$schema";
+import type { database } from "@packages/db";
+import { eq } from "@packages/db/drizzle";
+import { libraryTraffic } from "@packages/db/schema";
+import type { z } from "zod";
+
+type LibraryTrafficServiceInput = z.infer<typeof libraryTrafficQuerySchema>;
+type LibraryTrafficServiceOutput = typeof libraryTraffic.$inferSelect;
+
+export class LibraryTrafficService {
+  constructor(private readonly db: ReturnType<typeof database>) {}
+
+  async getLibraryTraffic(
+    input: LibraryTrafficServiceInput,
+  ): Promise<LibraryTrafficServiceOutput[]> {
+    return await this.db
```

Comment:
no need to await here, and if you make that change the function doesn't need to be async either